spec/rails_helper.rb

```diff
@@ -36,50 +18,22 @@
 end
 
 RSpec.configure do |config|
-  # add `FactoryBot` methods
-  config.include FactoryBot::Syntax::Methods
-
-  config.before(:all) do
-    FactoryBot.reload
-  end
-
   config.before(:suite) do
-    DatabaseCleaner.clean_with(:truncation)
-    DatabaseCleaner.strategy = :transaction
-  end
-
-  # start the transaction strategy as examples are run
-  config.around(:each) do |example|
-    DatabaseCleaner.cleaning do
-      example.run
+    FactoryBot.reload
+    config.include FactoryBot::Syntax::Methods
```

Comment:
looks like this shouldn't be here

Reply:
I believe config is related to upper block